### PR TITLE
Initial Ansible Inventory Generator

### PR DESCRIPTION
**Pull request by zehicle**
Creates a dynamic inventory file for use by Ansible.

This first pass that the file mainly shows that you can get the machines list in the correct format.

Future work would include:
1) passing parameters [DONE!]
2) grouping machines by profiles [DONE!]
3) setting values based on attribs [DONE!]

To test, simply run ./inventory and you'll see a list of machines from localhost with default password.

### CODE REVIEW

File: integrations/ansible/inventory.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  
+'''
+Usage: https://github.com/digitalrebar/provision/tree/master/integration/ansible
+
+example: ansible -i inventory.py all -a "uname -a"
+'''
+    
+def main():
+
+    # change these values to match your DigitalRebar installation
```
Comment (VictorLowther) [inline integrations/ansible/inventory.py]: These should default to appropriate values from the environment, like the CLI does.

Comment (galthaus) [inline integrations/ansible/inventory.py]: This:
    # change these values to match your DigitalRebar installation
    addr = os.getenv('RS_ENDPOINT', "https://127.0.0.1:8092")
    ups = os.getenv('RS_KEY', "rocketskates:r0cketsk8ts")
    arr = ups.split(":")
    user = arr[0]
    password = arr[1]

**Comment by zehicle** [comment]
@galthaus @VictorLowther added missing env variables

With addition of profile params, this is now functionally complete and ready for Ansible users!

**Comment by zehicle** [comment]
Rebased to master.
Added ability to support children grouping (as per k8s-cluster group need for Kubespray)

**Comment by zehicle** [comment]
DO NOT PULL THIS YET - Ansible updated the format to be YAML not INI.

**Comment by zehicle** [comment]
Latest changes to JSON format are WORKING on Ansible plays for Kubespray - this should be safe to pull

**Pull request merged**